sdks/go/pkg/beam/runners/dataflow/dataflowlib/job.go

```diff
@@ -345,5 +346,14 @@ func validateWorkerSettings(ctx context.Context, opts *JobOptions) error {
 		opts.WorkerZone = opts.Zone
 		opts.Zone = ""
 	}
+
+	numWorkers := opts.NumWorkers
+	maxNumWorkers := opts.MaxNumWorkers
+	if numWorkers < 0 {
```

Comment:
Should we should check maxNumWorkers >= 0 too?

Reply:
Sure!